include/flatbuffers/flatbuffers.h

```diff
@@ -2142,13 +2142,30 @@ struct TypeTable;
 // Signature of the static method present in each type.
 typedef TypeTable *(*TypeFunction)();
 
+// Key-value structure as used by an attribute
+struct AttributeKeyValue {
+  const char *key;
+  const char *value;
+};
+
+// Key-value pair array for storing custom attribute values.
+template<size_t count> struct AttributeList {
```

Comment:
this `count` template really doesn't have much use, and causes some unneeded typedeffing and casting below, let's remove it.